src/Event/TestSuite/Sorted.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Event\TestSuite;
+
+use PHPUnit\Event\Event;
+
+final class Configured implements Event
```

Comment:
Should be `Sorted` I think :slightly_smiling_face:

Reply:
Thank you, @rpkamp!